### PR TITLE
[nats-streaming] Release v0.10.0

### PR DESCRIPTION
**Pull request by kozlovic**
Details can be found [here](https://github.com/nats-io/nats-streaming-server/releases/tag/v0.10.0)

Signed-off-by: Ivan Kozlovic <ivan@synadia.com>

### CODE REVIEW

File: library/nats-streaming [hunk 1]
```diff
@@ -1,28 +1,28 @@
 Maintainers: Ivan Kozlovic <ivan@synadia.com> (@kozlovic)
 GitRepo: https://github.com/nats-io/nats-streaming-docker.git
-GitCommit: eaff03b1e4f88d0eb0eb476a8d1b36cac6fdf8df
+GitCommit: 5639c0377daecb06d162641d81c830fa5a204c00
 
 Tags: 0.9.2-linux, linux
 SharedTags: 0.9.2, latest
```
Comment (tianon) [inline library/nats-streaming]: If this is now 0.10, shouldn't the tags change to reflect that?

Comment (kozlovic) [reply library/nats-streaming]: Oh right, let me update the PR.. sorry..

**Comment by kozlovic** [comment]
@tianon So sorry about that, and thanks for the catch!!!

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 9731faa..9e62e14 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,7 +1,7 @@
-nats-streaming:0.9.2
-nats-streaming:0.9.2-linux
-nats-streaming:0.9.2-nanoserver
-nats-streaming:0.9.2-windowsservercore
+nats-streaming:0.10.0
+nats-streaming:0.10.0-linux
+nats-streaming:0.10.0-nanoserver
+nats-streaming:0.10.0-windowsservercore
 nats-streaming:latest
 nats-streaming:linux
 nats-streaming:nanoserver
diff --git a/_bashbrew-shared-tags b/_bashbrew-shared-tags
index b507b87..540bd6a 100644
--- a/_bashbrew-shared-tags
+++ b/_bashbrew-shared-tags
@@ -1,2 +1,2 @@
-nats-streaming:0.9.2 -- nats-streaming:linux, nats-streaming:nanoserver
+nats-streaming:0.10.0 -- nats-streaming:linux, nats-streaming:nanoserver
 nats-streaming:latest -- nats-streaming:linux, nats-streaming:nanoserver
diff --git a/nats-streaming_linux/nats-streaming-server b/nats-streaming_linux/nats-streaming-server
index de77db4..016a8ed 100755
Binary files a/nats-streaming_linux/nats-streaming-server and b/nats-streaming_linux/nats-streaming-server differ
diff --git a/nats-streaming_nanoserver/nats-streaming-server.exe b/nats-streaming_nanoserver/nats-streaming-server.exe
index 07dda07..8a69ba1 100755
Binary files a/nats-streaming_nanoserver/nats-streaming-server.exe and b/nats-streaming_nanoserver/nats-streaming-server.exe differ
diff --git a/nats-streaming_windowsservercore/nats-streaming-server.exe b/nats-streaming_windowsservercore/nats-streaming-server.exe
index 07dda07..8a69ba1 100755
Binary files a/nats-streaming_windowsservercore/nats-streaming-server.exe and b/nats-streaming_windowsservercore/nats-streaming-server.exe differ
```

</details>

**Comment by yosifkit** [comment]
Build test of #4472; 93caa76b; `amd64` (`nats-streaming`):

```console

$ bashbrew build nats-streaming:0.10.0-linux
Using bashbrew/cache:aeb237194a97dcf21008c168818acefb31dee68707e170d12f7dc5994bd6d0bc (nats-streaming:0.10.0-linux)
Tagging nats-streaming:0.10.0-linux
Tagging nats-streaming:linux
Tagging nats-streaming:0.10.0
Tagging nats-streaming:latest

$ test/run.sh nats-streaming:0.10.0-linux
testing nats-streaming:0.10.0-linux
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...image does not appear to contain "echo" -- assuming single-binary image
passed


$ bashbrew build nats-streaming:0.10.0-nanoserver
skipping "nats-streaming:0.10.0-nanoserver" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh nats-streaming:0.10.0-nanoserver
testing nats-streaming:0.10.0-nanoserver
	image does not exist!


$ bashbrew build nats-streaming:0.10.0-windowsservercore
skipping "nats-streaming:0.10.0-windowsservercore" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh nats-streaming:0.10.0-windowsservercore
testing nats-streaming:0.10.0-windowsservercore
	image does not exist!

```


The following images failed at least one test:

- `nats-streaming`:
  - `nats-streaming:0.10.0-nanoserver`
  - `nats-streaming:0.10.0-windowsservercore`

**Pull request merged**